code/_globalvars/traits.dm

```diff
@@ -69,12 +69,14 @@ GLOBAL_LIST_INIT(traits_by_type, list(
 		"TRAIT_AI_UNTRACKABLE" = TRAIT_AI_UNTRACKABLE,
 		"TRAIT_ELITE_CHALLENGER" = TRAIT_ELITE_CHALLENGER,
 		"TRAIT_SOAPY_MOUTH" = TRAIT_SOAPY_MOUTH,
-		"TRAIT_UNREVIVABLE" = TRAIT_UNREVIVABLE
+		"TRAIT_UNREVIVABLE" = TRAIT_UNREVIVABLE,
+		"TRAIT_CAN_BE_EATEN_BY_LIZARDS" = TRAIT_EDIBLE_BUG
```

Comment:
Ehm... doesn't seem related to this change. Unless you employed lizards to eat the bugs you found while testing

Reply:
I forgot to do this in my previous pr 💀